test/image/mocks/funnel_horizontal_stack_basic.json

```diff
@@ -0,0 +1,62 @@
+{
+    "data": [
+        {
+            "name": "plotly.js",
```

Comment:
Something looks off in the blue hover label path:

![Peek 2019-04-29 17-55](https://user-images.githubusercontent.com/6675409/56929612-1ae86a80-6aa8-11e9-94fe-b4c5529892ba.gif)


This is probably not related to your work, can you confirm that `bar` traces show the same problem?

Reply:
Copied from @alexcjohnson message.
<i>Here’s the original comment about rotated & shifted hover labels - the issue got closed without spinning out a new issue for this part that’s still open </i> https://github.com/plotly/plotly.js/issues/3050#issuecomment-424508921

Reply:
@archmoj #3050 is closed. Can you open a new issue?

Reply:
Open this one: https://github.com/plotly/plotly.js/issues/3838.

Reply:
Thanks!